static/js/ducks/source.js

```diff
@@ -266,7 +289,7 @@ export function getCommentOnSpectrumAttachmentPreview(spectrumID, commentID) {
 
 export function fetchSource(id, actionType = FETCH_LOADED_SOURCE) {
   return API.GET(
-    `/api/sources/${id}?includeComments=true&includeColorMagnitude=true&includeThumbnails=true&includePhotometryExists=true&includeSpectrumExists=true`,
+    `/api/sources/${id}?includeComments=true&includeColorMagnitude=true&includeThumbnails=true&includePhotometryExists=true&includeSpectrumExists=true&includeLabellers=true`,
```

Comment:
Is there a way to move all these URL parameters into a parameter object to pass like we would do for python? This url is getting a bit long...

Reply:
Yeah I think fixed.